2023/gaussian-convolution/german/sentence_translations.json

```diff
@@ -164,7 +164,7 @@
  {
   "input": "And if you want to consider distributions that aren't necessarily centered at zero, you would also throw another parameter, mu, into the exponent like this.",
   "model": "nmt",
-  "translatedText": "Und wenn Sie Verteilungen berücksichtigen möchten, die nicht unbedingt bei Null zentriert sind, würden Sie auch einen weiteren Parameter, mu, wie folgt in den Exponenten einfügen.",
+  "translatedText": "Und wenn Sie Verteilungen berücksichtigen möchten, die nicht unbedingt bei Null zentriert sind, würden Sie auch einen weiteren Parameter, mü, wie folgt in den Exponenten einfügen.",
```

Comment:
I chose this over something orthographically correct because I was afraid that some text to speach system might again turn it into english pronunciation. Perhaps the µ from the keyboard is the best solution here?

Reply:
I already thought that this might be your motivation for the _mü_, and I'm no TTS expert, but my reasoning was that it's probably not good to adapt to flawed TTS systems in this way, because if you want to use the translation also for subtitles, for example, then they should be orthographically correct, and also future TTS systems maybe don't have this problem anymore. I also thought about choosing μ over _my_, and the only reason I didn't was that it was mu and not μ in the original, and maybe there is a reason for that. But otherwise, I would have preferred μ too.